From a164a18fef1dcc6e15ee8f7d3b603b61bb2e1fb4 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 22 Nov 2024 11:52:52 +0100
Subject: [PATCH 1364/1695] dts: stm32: enable exti1 and exti2 on stm32mp211

Both EXTI devices should be enabled to apply the RIF configuration
and to allow clients to access to them.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 2dbaa71121aa158ee7235511772395b1ae5e7e4a dts: stm32: enable exti1 and exti2 on stm32mp211]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I87809b8ed67878baf818ad0b9f008559a98a42a7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/420308
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 0d6966e6c..8e42d370b 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -2579,7 +2579,6 @@
 			#interrupt-cells = <2>;
 			#address-cells = <0>;
 			reg = <0x44220000 0x400>;
-			status = "disabled";
 		};
 
 		syscfg: syscon@44230000 {
@@ -2708,7 +2707,6 @@
 			#interrupt-cells = <2>;
 			#address-cells = <0>;
 			reg = <0x442d0000 0x400>;
-			status = "disabled";
 		};
 
 		rtc: rtc@46000000 {
-- 
2.47.2

